### DISCUSSION TITLE
Introduction of Github templates

### DISCUSSION THREAD

**Discussion by rartych** [camaraproject/IdentityAndConsentManagement#38] [General]
**Problem description**
Github templates for issues and pull requests were configured for this repository.

**Expected action**
You can use it for new issues or open blank issue. Feel free to edit texts and proposed structure according to your needs.

**Additional context**
More info is included in [Commonalities documentation](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/Issue%20and%20PR%20template%20Howto.md).

**Comment by jpengar** [comment]
@rartych This issue has been already addressed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/19, hasn't it?

**Comment by hdamker** [comment]
@rartych maybe you want to convert this issue into an announcement within "discussions". (https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion)
